test/verify/check-users

```diff
@@ -298,6 +298,30 @@ class TestAccounts(testlib.MachineCase):
             b.click("#account-confirm-delete-dialog footer button.pf-m-danger.apply")
             b.wait_not_in_text('#accounts-list', "Robert Robertson")
 
+        # test logout other user
+        createUser(
+            browser=b,
+            machine=m,
+            user_name="paul",
+            real_name="Paul Robertson",
+            password=good_password,
+            locked=False,
+            force_password_change=False,
+            custom_shell="/bin/bash",  # CoreOS defaults to /bin/true
+        )
+        b2 = self.new_browser(m)
+        b2.login_and_go("/system", user="paul", password=good_password)
+        b.wait_in_text("#accounts-list tbody tr:contains('paul')", "Logged in")
+
+        performUserAction(b, 'paul', 'Log user out')
```

Comment:
This triggered a strong déjà vu. But on closer inspection we have an [existing test](https://github.com/cockpit-project/cockpit/blob/5d3ac3713643fc3f4e9190a572595d73833a897b/test/verify/check-users#L635) for the "Terminate session" . But this is a different button. :+1:

Reply:
Hmm interesting, my idea was to cover [this](https://cockpit-logs.us-east-1.linodeobjects.com/pull-21389-0c41f814-20250109-101243-fedora-41-devel/Coverage/pkg/users/logout-account-dialog.js.gcov.html).

Which reminds me, we should check the title.

Reply:
And this is now covered https://cockpit-logs.us-east-1.linodeobjects.com/pull-21562-385717dc-20250130-132724-fedora-41-devel/Coverage/pkg/users/index-sort-l.html